service/src/main/java/io/techery/janet/AsyncActionsRosterBase.java

```diff
@@ -9,7 +10,10 @@
     protected final ConcurrentHashMap<String, List<Class>> map = new ConcurrentHashMap<String, List<Class>>();
 
     List<Class> getActionClasses(String event) {
-        return map.get(event);
+        if (map.containsKey(event)) {
+            return map.get(event);
```

Comment:
map.get does the same job contains does. So it's better `var list = map.get(); if (list != null)...`